andela/k2-cfh
# feature(design game page): Re-Design UI for the actual gaming screen

#### What does this PR do?

Re-Design UI for the gaming screen

#### Description of Task to be completed?

- redesign the UI for the actual gaming screen

#### How should this be manually tested?
- Clone the repository
- Run `git checkout feature/153009262/re-design-gaming-screen`
- Run `npm start`
- Run `npm test` to start server
- Click the start playing button to see the new UI

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

- story type: feature
- story id: 153009262
- story title: Re-Design UI for the actual gaming screen

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2017-12-09 at 10 12 45 pm" src="https://user-images.githubusercontent.com/14099353/33799543-2ce37314-dd2e-11e7-98a1-06addae890dc.png">

#### Questions:

N/A
